include/deal.II/lac/petsc_parallel_sparse_matrix.h

```diff
@@ -408,6 +408,26 @@ namespace PETScWrappers
        */
       IndexSet locally_owned_range_indices() const;
 
+      /**
+       * Perform the matrix-matrix multiplication $C = AB$, or,
+       * $C = A \text{diag}(V) B$ given a compatible vector $V$.
+       *
+       * This function calls <code>MatrixBase::mmult</code> to do the actual work.
```

Comment:
If you write this as `MatrixBase::mmult()`, then you don't need the `<code>...</code>` markup -- doxygen will automatically link to that function.

Same below of course.

Reply:
Thanks, that will be fixed in the next round.